### PR TITLE
Add my testimonials

### PR DESCRIPTION
**Pull request by Terrocraft**
I made a testimonial for the website/cloud.

### CODE REVIEW

File: testimonials/testimonials.json [hunk 1]
```diff
@@ -10,7 +10,19 @@
         "x": "egyboy_",
         "bsky": "egyboyy.bsky.social",
         "github": "egyboyy"
+      },
+    {
```
Comment (FllipEis) [inline testimonials/testimonials.json:14]: Your formatting is wrong. Please check that with some json format checker ([this one](https://jsonformatter.curiousconcept.com) for example)

File: testimonials/testimonials.json [hunk 2]
```diff
@@ -10,7 +10,19 @@
...
+      "username": "Terrocraft",
+      "role": "Simplecloud user and idea contributor",
+      "skinUrl": "https://raw.githubusercontent.com/theSimpleCloud/simplecloud-manifest/refs/heads/main/testimonials/skins/terro.png",
+      "comment": "The cloud is really simple and user-friendly to use. You can do a lot with the API and it is relatively simple. I think the new cli is particularly good and innovative.",
+      "userSince": "14. Feb 2024",
```
Comment (FllipEis) [inline testimonials/testimonials.json:19]: Use either something like `Winter 2024` or `14th Feb. 2024`

**Review by FllipEis — APPROVED** [review]

File: testimonials/testimonials.json (hunk 2 above)
Comment (Terrocraft) [reply testimonials/testimonials.json:19]: 14th Feb. 2024

File: testimonials/testimonials.json
Comment (FllipEis) [inline testimonials/testimonials.json:1]: Your formatting is still broken

**Review by FllipEis — CHANGES_REQUESTED** [review]

File: testimonials/testimonials.json
Comment (Terrocraft) [reply testimonials/testimonials.json:1]: should I add a section or subtract the one from egyboy_?

**Pull request closed without merging**